### ISSUE TITLE
Validation accuracy hovering around 50%

### ISSUE THREAD

**Issue by nirajpandkar** [nirajpandkar/keras-cats-dogs#1]

**Comment by nirajpandkar** [comment]
Solved.

Include preprocess_input in ImageDataGenerator constructor.
